chore: Change ints to 'number' for ProgressEvent properties (#39574)
mdn/content: files/en-us/web/api/progressevent/total/index.md

{{APIRef("XMLHttpRequest API")}}{{AvailableInWorkers}}

> The **`ProgressEvent.total`** read-only property is a double-precision floating point value

Author: ```suggestion
The **`ProgressEvent.total`** read-only property is a number
```

## Value

> A `double`.

Author: ```suggestion
A floating-point number.
```

> A floating-point number.

Author: ```suggestion
A number.
```
